docs/reference/elastic-search.md

```diff
@@ -106,3 +109,48 @@ WIRE_VERSION=<VERSION_YOU_ARE_DEPLOYING>
 1. Redeploy brig without `elasticsearch.additionalWriteIndex` and with `elasticsearch.index` set to the name of new index
 
 Now you can delete the old index.
+
+## Recreate an index
```

Comment:
```suggestion
## Recreate an index (simpler but requires downtime!)
```

Reply:
I added the "simpler" part in documentation for migrating, as this being simpler on top level is a bit confusing.